nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
Similar to this: https://github.com/scala/scala/blob/v2.12.1/src/library/scala/util/Properties.scala#L145-L152